test/Microsoft.OpenApi.Readers.Tests/ReferenceService/ConvertToOpenApiReferenceV3Tests.cs

```diff
@@ -1,145 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
-using Microsoft.OpenApi.Models;
-using Microsoft.OpenApi.Reader;
-using Microsoft.OpenApi.Reader.V3;
-using Xunit;
-
-namespace Microsoft.OpenApi.Readers.Tests
```

Comment:
why are we removing a bunch of unit tests?

Reply:
thanks for the contextualization here. This PR is large. I simply wanted to ensure we were not removing unit tests for code that remains simply to make our lives easier: coverage should remain the same or improve :)